src/css/inputs.css

```diff
@@ -0,0 +1,54 @@
+@layer utilities {
+    .radio-indicator {
```

Comment:
a few things for this file as it's hard to read:
- can you add a comment above the start of the elements it targets, e.g. `/* Radio Buttons */` and `/* Checkboxes */` to make it easier to figure out where certain styles are
- I'd try to avoid the `[0.15rem]` values as they result in partial pixels, better to check designs and if we should use 2 or 3px borders instead
- there is some repetition with the various classes, I'll highlight those in separate comments